assets/translations/en.json

```diff
@@ -287,7 +286,8 @@
     "normal": "Normal",
     "shizuku": "Shizuku",
     "root": "Root",
-    "shizukuBinderNotFound": "Shizuku is not running",
+    "shizukuBinderNotFound": "Сompatible Shizuku service wasn't found",
+    "tryUseSystemFont": "Try to use a system font",
```

Comment:
I think this should say '_the_ system font` instead of '_a_ system font'.

Also in what situations will this not work? It worked on an Android 14 emulator without Shizuku. Does it depend on the Android version? If so we can hide the menu item on older versions.